### ISSUE TITLE
Impossible to detect the hidden html5 player and download anime streaming video

### ISSUE THREAD

**Issue by trimechee** [chandler-stimson/live-stream-downloader#56]
Hello, thank you again so much for this awesome great downloader ! 💯 🥇

I noticed it is impossible to detect the html5 video player in this anime streaming site and therefore we cannot download this videos :

https://aniwave.to/watch/erased.kkw/ep-6

and

https://kissanime.com.ru/Anime/boku-dake-ga-inai-machi-dub.22213/Episode-003?id=131684

maybe the site hides the html5 video player against extensions and download manager....I even tried for the kissaanime site with Filecenitpede, FDM, coc coc browser, downthemall for the palemoon browser but these softwares did not succeed in detecting and download the videos from kissanime website and we hope this issue will be resolved, thank you very much :)

**Comment by chandler-stimson** [comment]
Same as https://github.com/chandler-stimson/live-stream-downloader/issues/58.

**Comment by trimechee** [comment]
Yes Mr @chandler-stimson, and it seems it will very difficult hard complex to support download from kissanime website beacause i talk to Coc Coc browser team member to add support downloadinfg from kissanime (Oserver video player) and he tells me :

 "We're sorry to inform you that Coc Coc does not support downloads on the site: 
  
https://kissanime.com.ru/Anime/boku-dake-ga-inai-machi-dub.22213/Episode-004?id=131687&s=oserver 

for this reason: This site uses the Hydrax server so we won’t support it. "


Also the powerful free download manager , neat download manager can't detect videos and download from hydrax server in kissanime website.....but the developer of open source  FileCentipede Download Manager told me that it is possible to add support for downloading FROM kissanime (hydrax server)....

https://github.com/filecxx/FileCentipede

We remain hopeful that our beloved Live-stream-downloader developed by the great Mr @chandler-stimson 🥇 could one day support kissanime website please so no more need for a download manager which remains active in the background.... thank you so much :)
